### PR TITLE
Add wannabe is good (#2)

### PR DESCRIPTION
**Pull request by devFallingstar**
Add wannabe is good (#2)

### CODE REVIEW

File: wannabe.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+wannabe is good
```
Comment (XIU-kr) [inline wannabe.txt:1]: wannabe is bad --

**Review by XIU-kr — CHANGES_REQUESTED** [review]

File: wannabe.txt (hunk 1 above)
Comment (devFallingstar) [reply wannabe.txt:1]: WTF wannabe is good!!!

Comment (XIU-kr) [reply wannabe.txt:1]: ok sry wannabe is good!

**Review by XIU-kr — APPROVED** [review]
LGTM

**Review by XIU-kr — APPROVED** [review]

**Review by XIU-kr — APPROVED** [review]

**Pull request merged**